CHANGELOG.adoc

```diff
@@ -11,6 +11,7 @@ A release with an intentional breaking changes is marked with:
 (https://github.com/tupini07[@tupini07])
 * https://github.com/clj-commons/etaoin/issues/566[#566]: Recognize `:driver-log-level` for Edge
 * bump all deps to current versions
+* https://github.com/clj-commons/etaoin/issues/602[#602]: Document all :fn/* query pseudo-functions in a definitive list
```

Comment:
Awesome @drg! Can you move this under the docs heading below?

Reply:
Ah, yes, will do.